core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -569,7 +569,7 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
 
   protected def consumerGroupHeartbeat(
     groupId: String,
-    memberId: String = "",
+    memberId: String = Uuid.randomUuid().toString,
```

Comment:
Since we are upgrading the `ConsumerGroupHeartbeat`, the `memberId` cannot be empty or null.
It would be nice to test the older version of the protocol, but I'm not quite sure how to achieve that.

Reply:
Hi @dajac 
I have a question about test in server-side. Could you please take a look ?
Many thanks 😺

Reply:
You can pass the version and use it. See `commitOffset` in this file as an example. We should also update ConsumerGroupHeartbeatRequestTest.scala btw.